erpnext/accounts/doctype/promotional_scheme/promotional_scheme.json

```diff
@@ -1,1381 +1,332 @@
 {
- "allow_copy": 0,
- "allow_events_in_timeline": 0,
- "allow_guest_to_view": 0,
  "allow_import": 1,
  "allow_rename": 1,
  "autoname": "Prompt",
- "beta": 0,
  "creation": "2019-02-08 17:10:36.077402",
- "custom": 0,
- "docstatus": 0,
  "doctype": "DocType",
- "document_type": "",
  "editable_grid": 1,
  "engine": "InnoDB",
+ "field_order": [
+  "section_break_1",
+  "disable",
```

Comment:
- Move Disable to the above section

Reply:
Its already moved to above section .
![image](https://user-images.githubusercontent.com/80836439/118626848-d5ab4880-b7e8-11eb-9ecd-fd21c15b8ea5.png)